mne/preprocessing/artifact_detection.py

```diff
@@ -462,6 +463,11 @@ def annotate_break(raw, events=None,
             logger.info(f'Ignoring annotations with descriptions starting '
                         f'with: {", ".join(ignore)}')
     else:
+        # XXX: I think a bug is hiding here with regard to first_samp
+        #      We should extend the test to work on events and see if
+        #      it breaks here.
+        #      "annotations_from_events" takes a "first_samp" param,
+        #      but this is not being passed here!
```

Comment:
I pushed a commit that shows that `annotate_nan` still fails.

AND I think this :point_up: is a hint that for `annotate_break` we might be missing something as well.

Reply:
update: pushed a commit that reveals that annotate_break is still broken

Reply:
```suggestion
```